namespaces/live.cloud-platform.service.justice.gov.uk/migration-poc-spike-dev/resources/cross-iam-role-sa.tf

```diff
@@ -0,0 +1,43 @@
+module "irsa" {
+  eks_cluster_name = "live.cloud-platform.service.justice.gov.uk"
```

Comment:
Can you add the variable to variables.tf
 `variable "eks_cluster_name" {
  description = "The name of the eks cluster to retrieve the OIDC information"
}`

The value for `eks_cluster_name` is provided from the pipeline,